content/scalability/docs/control-plane.md

```diff
@@ -240,4 +244,50 @@ spec:
 
 - This FlowSchema captures all list event calls made by service accounts in the default namespace.
 - The matching precedence 8000 is lower than the value of 9000 used by the existing service-accounts FlowSchema so these list event calls will match list-events-default-service-accounts rather than service-accounts.
-- We're using the catch-all PriorityLevelConfiguration to isolate these requests. This bucket only allows 13 inflight requests to be used by these long-running list event calls. Pods will start to receive 429s as soon they try to issue more than 13 of these requests concurrently. 
+- We're using the catch-all PriorityLevelConfiguration to isolate these requests. This bucket only allows 13 inflight requests to be used by these long-running list event calls. Pods will start to receive 429s as soon they try to issue more than 13 of these requests concurrently.
+
+## Use Shared Informers 
+
+When building controllers and automation that integrate with the Kubernetes API you will often need to get information from Kubernetes resources. If you poll for these resources regularly it can cause a significant load on the API server.
+
+Using an [informer](https://pkg.go.dev/k8s.io/client-go/informers) from the client-go library will give you benefits of watching for changes to the resources based on events instead of polling for changes. Informers further reduce the load by using shared cache for the events and changes so multiple controllers watching the same resources do not add additional load.
+
+Controllers should avoid polling cluster wide resources without labels and field selectors especially in large clusters. Each un-filtered poll requires a lot of unnecessary data to be sent from etcd through the API server to be filtered by the client. By filtering based on labels and namespaces you can reduce the amount of work the API server needs to perform to fullfil the request and data sent to the client.
+
+## Optimize Kubernetes API usage
+
+When calling the Kubernetes API with custom controllers or automation it's important that you limit the calls to only the resources you need. Without limits you can cause unneeded load on the API server and etcd.
```

Comment:
@rothgar same comment as above. Pasting it below for reference -
"We know this (list vs watch) is a major cause of API server latency/unresponsiveness from the customer tickets we get. Can we change the phrasing a bit to make the reader treat this recommendation with more urgency/seriousness. Maybe a line or two on the ramifications of "it's important that..." and how it can lead to API server latency/slowness/unresponsiveness will help. Also, perhaps we can callout that this is a common cause of various API server issues/latencies that we see .